tests/StlTests.cpp

```diff
@@ -50,6 +51,42 @@ TEST_CASE("StlTests::TestSecureAllocator", "[stl][secure_allocator]")
     }
 }
 
+// MockMethod<void(std::allocator<T>::*)(T*, size_t)>

```

Comment:
I was debating adding support for mocking member functions, but never got around to it. I don't think it would be _too_ hard, but we already have decent enough validation of the `secure_deleter` type that I'm going to punt it for now.